app/uk/gov/hmrc/taxhistory/connectors/CitizenDetailsConnector.scala

```diff
@@ -21,21 +21,22 @@ import play.api.libs.json.JsValue
 import uk.gov.hmrc.domain.{Nino, SaUtr}
 import uk.gov.hmrc.http._
 import uk.gov.hmrc.play.bootstrap.http.HttpClient
-import uk.gov.hmrc.play.http.logging.MdcLoggingExecutionContext.fromLoggingDetails
+import uk.gov.hmrc.taxhistory.config.AppConfig
 import uk.gov.hmrc.taxhistory.metrics.{MetricsEnum, TaxHistoryMetrics}
 import uk.gov.hmrc.taxhistory.utils.Retry
 
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
 
 class CitizenDetailsConnector @Inject()(val http: HttpClient,
                                         val metrics: TaxHistoryMetrics,
-                                        @Named("citizen-details-base-url") val baseUrl: String,
-                                        @Named("citizen-details") val withRetry: Retry)  extends ConnectorMetrics {
+                                        config: AppConfig,
+                                        @Named("citizen-details") val withRetry: Retry)
```

Comment:
I know the '@Named' thing isn't something you added, but is it actually used?

Reply:
At the minute that's how it's pulling in the "withRetry" function that is being used further down. Once I've done the above changes though it can just be pulled in with the rest of the config